python/social_bot/envs/play_ground.py

```diff
@@ -194,6 +194,7 @@ def reset(self):
         self._steps_in_this_episode = 0
         self._world.reset()
         self._teacher.reset(self._agent, self._world)
+        self._world.step(self._sub_steps)
```

Comment:
This should be after line 199 because teacher may make some change to the environment (e.g. randomly move objects)